### PR TITLE
Added translation of alerts, added Russian translation

### PR DESCRIPTION
**Pull request by giwih**
you can translate “Pick color” and “Reload the page after applying themes” as well as all dialog boxes:
image
image

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ if (!localStorage.getItem('alertShown')) {
     setTimeout(() => {
         const isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
         const message = isMac
-            ? 'Press Cmd + Shift + B to show the bookmarks bar.'
-            : 'Press Ctrl + Shift + B to show the bookmarks bar.';
+            ? translations[currentLanguage]?.Macbookmarks || translations['en'].Macbookmarks
```
Comment (prem-k-r) [inline script.js:16]: This one did not need to be translated, as it is shown right away when the extension is installed. At that time, the default language is English.

**Review by prem-k-r — CHANGES_REQUESTED** [review]

**Review by prem-k-r — APPROVED** [review]

**Comment by prem-k-r** [comment]
If you don't mind, could you please fix the hover text for `#menuCloseButton`? It seems to have stopped working.  
What you did for the hover text in `.themingStuff` can be replicated for this as well.

**Comment by prem-k-r** [comment]
@XengShi 
image

**Comment by XengShi** [comment]
> @XengShi image

I noticed that tooo

**Comment by XengShi** [comment]
> If you don't mind, could you please fix the hover text for `#menuCloseButton`? It seems to have stopped working. What you did for the hover text in `.themingStuff` can be replicated for this as well.

hover on close menu button is working for me!

**Comment by prem-k-r** [comment]
> > If you don't mind, could you please fix the hover text for `#menuCloseButton`? It seems to have stopped working. What you did for the hover text in `.themingStuff` can be replicated for this as well.
> 
> hover on close menu button is working for me!

odd lol, working here too https://xengshi.github.io/materialYouNewTab/
without changing anything

**Comment by prem-k-r** [comment]
image
but not working on my pc

**Comment by prem-k-r** [comment]
> > > > If you don't mind, could you please fix the hover text for `#menuCloseButton`? It seems to have stopped working. What you did for the hover text in `.themingStuff` can be replicated for this as well.
> > > 
> > > 
> > > hover on close menu button is working for me!
> > 
> > 
> > odd lol, working here too [xengshi.github.io/materialYouNewTab](https://xengshi.github.io/materialYouNewTab/) without changing anything
> 
> If you change the theme, the close button will always be “close” regardless of language?

yes, it's not always showing the translated text for close

**Comment by XengShi** [comment]
> > > > If you don't mind, could you please fix the hover text for `#menuCloseButton`? It seems to have stopped working. What you did for the hover text in `.themingStuff` can be replicated for this as well.
> > > 
> > > 
> > > hover on close menu button is working for me!
> > 
> > 
> > odd lol, working here too https://xengshi.github.io/materialYouNewTab/ without changing anything
> 
> If you change the theme, the close button will always be “close” regardless of language?

I think we need to use the same method that we are using for translating other elements.

**Comment by prem-k-r** [comment]
> > > > > If you don't mind, could you please fix the hover text for `#menuCloseButton`? It seems to have stopped working. What you did for the hover text in `.themingStuff` can be replicated for this as well.
> > > > 
> > > > 
> > > > hover on close menu button is working for me!
> > > 
> > > 
> > > odd lol, working here too [xengshi.github.io/materialYouNewTab](https://xengshi.github.io/materialYouNewTab/) without changing anything
> > 
> > 
> > If you change the theme, the close button will always be “close” regardless of language?
> 
> I think we need to use the same method that we are using for translating other elements.

`Close` text is in css file.. html and js origin strings have different approaches

**Comment by XengShi** [comment]
Screenshot 2024-12-08 at 22-31-12 Validation Results for Material-You-NewTab-v2 5 zip Developer Hub Add-ons for Firefox

@prem-k-r Whenever you get some free time and would like to help find a solution for these warnings, you are more than welcome to look into them.

**Comment by prem-k-r** [comment]
Ok, I'll check out tomorrow..

**Pull request merged**